internal/mode/static/state/graph/gatewayclass.go

```diff
@@ -87,7 +86,7 @@ func validateGatewayClass(gc *v1beta1.GatewayClass, npCfg *ngfAPI.NginxProxy) er
 			return field.Required(path.Child("namespace"), "parametersRef.namespace must be specified for NginxProxy")
 		}
 
-		if npCfg == nil {
+		if npCfg == nil || npCfg.Source == nil {
```

Comment:
can npCfg.Source be nil when npCfg is not nil?

Reply:
No, I've removed the code in question